liquibase-standard/src/main/java/liquibase/plugin/AbstractPluginFactory.java

```diff
@@ -93,7 +93,8 @@ public synchronized void register(T plugin) {
      * If the instances should not be treated as singletons, clone the objects before returning them from {@link #getPlugin(Object...)}.
      */
     protected synchronized Collection<T> findAllInstances() {
-        if (this.allInstances == null) {
+        Boolean osgiPlatform = Scope.getCurrentScope().get(Scope.Attr.osgiPlatform, Boolean.class);
```

Comment:
I rolled back this osgi logic at master, seems this file should not be included in this pr?